raster3d/r3.in.v5d/v5d.c

```diff
@@ -1334,8 +1340,8 @@ static int read_comp_header(int f, v5dstruct *v)
 
                 /* skip ahead by 'gridsize' bytes */
                 if (lseek(f, gridsize, SEEK_CUR) == -1) {
-                    printf("Error:  Unexpected end of file, ");
-                    printf("file may be corrupted.\n");
+                    G_warning(_("Error:  Unexpected end of file, "));
+                    G_warning(_("file may be corrupted.\n"));
```

Comment:
I don't see a reason why the original lines are split except for making the lines shorter. It is one message, so it should be one G_warning call.

Reply:
Done changes